envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadata.kt

```diff
@@ -569,6 +592,20 @@ data class DomainPatternDependency(
     override fun useSsl() = DEFAULT_HTTPS_POLICY
 }
 
+data class TagDependency(
+    val tag: String,
+    val settings: DependencySettings = DependencySettings()
+) : Dependency {
+    companion object {
+        private const val DEFAULT_HTTP_PORT = 80
+        private const val DEFAULT_HTTPS_POLICY = false
+    }
+
+    override fun getPort() = DEFAULT_HTTP_PORT
```

Comment:
what do we need? This `TagDependency` class?

Reply:
Default Port

Reply:
I'm not sure how it works. But Virtual Listeners are created based on port defined in dependency, so it is required. I used same port as in ServiceDependency, because TagDependency are based on services